### DISCUSSION TITLE
Support for PKCE

### DISCUSSION THREAD

**Discussion by DragonLi-Mi** [apache/apisix#6323] [Q&A]
Is there any support for PKCE for OAuth2？Where can I config the code_challenge?

**Comment by tao12345666333** [comment]
It is not currently supported. If you have this requirement, you can create an issue for discussion. This is easier to implement

**Comment by rommelandrea** [comment]
@DragonLi-Mi i see that version 2.15 add this feature.
Here you can find the release notes https://apisix.apache.org/blog/2022/07/29/release-apache-apisix-2.15/
